src/components/UI/FormFields/InputField/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { TextField, TextFieldProps } from "@mui/material";
+import { FieldError } from "../FieldError";
+
+type IProps = {
+  field: any;
```

Comment:
Why the field is any? What is the purpose of that prop?

Reply:
its used in a later commit

Reply:
Can we type it?

Reply:
i checked yesterday, its the only way to infer this type. there is no way to extend with interface

Reply:
Fixed